adapters/ds/sinks/always_on_rtsp/__main__.py

```diff
@@ -217,6 +235,8 @@ def build_input_pipeline(
     gst_source_elements = add_elements(pipeline, source_elements, factory)
     gst_sink_elements = add_elements(pipeline, sink_elements, factory)
     savant_rs_video_demux = gst_source_elements[-1]
+    if config.video_pipeline is not None:
```

Comment:
Method `properties.update` in `NvDsPipeline` doesn't accept `VideoPipeline` since it's not a supported primitive type.
https://github.com/insight-platform/Savant/pull/367/files#diff-d45dbd7b97bdca3af480187b1127a53b934c84949df0f959ea14490730fd1871R117

So I've split setting of `savant_rs_video_demux` as in `NvDsPipeline`. But looks like we can set `VideoPipeline` as a property in the `PipelineElement` constructor. Fixed it.